### PR TITLE
Move Avatar size constants out of MSFAvatarSize

### PR DESCRIPTION
**Pull request by huwilkes**
### Platforms Impacted
- [x] iOS
- [ ] macOS

### Description of changes

Moving the size out of the enum as requested in https://github.com/microsoft/fluentui-apple/pull/919#discussion_r810401567

There were a few places where we would try to get the size of an Avatar by going through the MSFAvatarSize rather than the Avatar itself. 
One of those places was in some constraints for the LargeTitleView (added in https://github.com/microsoft/fluentui-apple/pull/535), but that seems to have no problems resizing without them.

### Verification

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| AvatarSize_Group_before | AvatarSize_Group_after |
| AvatarSize_PersonaButton_before | AvatarSize_PersonaButton_after |
| AvatarSize_LargeTitle_before | AvatarSize_LargeTitle_after |

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] iPad [Pointer interaction](https://developer.apple.com/documentation/uikit/pointer_interactions)
- [ ] [SwiftUI](https://developer.apple.com/tutorials/swiftui) consumption (validation or new demo scenarios needed)
- [ ] Objective-C exposure (provide it only if needed)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/fluentui-apple/pull/934)

### CODE REVIEW

File: ios/FluentUI/AvatarGroup/AvatarGroup.swift [hunk 1]
```diff
@@ -132,7 +132,7 @@ public struct AvatarGroup: View, ConfigurableTokenizedControl {
         let isStackStyle = tokens.style == .stack
 
         let interspace: CGFloat = tokens.interspace
-        let imageSize: CGFloat = tokens.size.size
+        let imageSize: CGFloat = avatarViews[0].contentSize
```
Comment (sophialee0416) [inline ios/FluentUI/AvatarGroup/AvatarGroup.swift:135]: Is it ok to access the size like this? What if the avatarViews was empty?

Comment (huwilkes) [reply ios/FluentUI/AvatarGroup/AvatarGroup.swift]: That's fair. I've changed it to make the overflow avatar earlier so we can get the content size from that. With the tokens just being stored in the actual control now, there's not really a good way to get the size without going through an actual avatar.

**Review by sophialee0416 — APPROVED** [review]

File: ios/FluentUI/Navigation/Views/LargeTitleView.swift [hunk 2]
```diff
@@ -58,9 +58,6 @@ class LargeTitleView: UIView {
         }
     }
 
-    var avatarHeightConstraint: NSLayoutConstraint?
```
Comment (harrieshin) [inline ios/FluentUI/Navigation/Views/LargeTitleView.swift:61]: what was the original change/commit/pr? what was the bug?

Comment (harrieshin) [reply ios/FluentUI/Navigation/Views/LargeTitleView.swift:61]: and actually mention it in your pr and commit message.

Comment (huwilkes) [reply ios/FluentUI/Navigation/Views/LargeTitleView.swift:61]: https://github.com/microsoft/fluentui-apple/pull/919#discussion_r810401567
There wasn't really a good way to get the size to the LargeTitleView, so I checked if it worked without the constraints, and it did.

**Pull request merged**